smg-automotive/components-pkg
# feat: tab component

References [VSST-479](https://autoricardo.atlassian.net/browse/VSST-479?atlOrigin=eyJpIjoiYjU4ZTRkZTFhNDQ3NDAzYzk3NDdhZjE3YTNjMzg5NjUiLCJwIjoiaiJ9)

## Motivation and context

For the first step of the insertion process we need to use Tabs
[Here is the design](https://www.figma.com/file/CHU4fpMfXpiAW4qNFzi9y1/11.-AS24---Web---Components?node-id=1862%3A59673&t=ButXzNCT5VJmRiQg-0)

## Before

No tab components

## After

Tab component

## How to test

Please make sure that the component works as intended and matches the design, [here](https://marine-vsst-479-components-pkg.branch.autoscout24.dev/?path=/story/components-navigation-tabs--overview).
Note: the selected border bottom can be used as provided by chakra: 
![dc0d6fe4-d633-451f-be85-d58cca4b6fbb](https://user-images.githubusercontent.com/52455155/210519109-7150e816-9dd5-4b62-9382-e70e35c34b82.png)

Thank you


[VSST-479]: https://autoricardo.atlassian.net/browse/VSST-479?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[VSST-479]: https://autoricardo.atlassian.net/browse/VSST-479?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
